src/nginxconfig/generators/conf/nginx.conf.js

```diff
@@ -47,6 +47,12 @@ export default (domains, global) => {
         worker_connections: 65535,
     };
 
+    // Modules!
+    config.http.push(['# Load modules', '']);
+    config.http.push(['include', [
+        global.tools.modularizedStructure.computed ? `${global.nginx.nginxConfigDirectory.computed.replace(/\/+$/, '')}/modules-enabled/*` : '',
+    ].filter(x => x.length)]);
```

Comment:
```suggestion
    config['# Load modules'] = '';
    config.include = `${global.nginx.nginxConfigDirectory.computed.replace(/\/+$/, '')}/modules-enabled/*`;
```
This is outside the HTTP context, so you'll just want to mutate the main config object. :)

I'm also not sure why the modularized structure toggle for the nginxconfig output would affect whether core nginx modules need to be loaded or not? I feel they will always need to be loaded?